### PR TITLE
MRP-459 Create skeleton

### PR DESCRIPTION
**Pull request by andrewi-wd**

### CODE REVIEW

File: docker/.env.ci [hunk 1]
```diff
@@ -0,0 +1,30 @@
+NODE_ENV=development
+APP_NAME=app
+CLUSTER=conn
+GIT_REPO_NAME="ce-connector-cotentful"
```
Comment (m-evans-1) [inline docker/.env.ci:4]: 🟠 Spelling mistake `cotentful`. Same in `.env.default`

**Review by m-evans-1 — APPROVED** [review]
🚀

File: prisma/schema.prisma [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider        = "prisma-client-js"
```
Comment (kibertoad) [inline prisma/schema.prisma:5]: there was a decision not to use foreign kdys, can you switch to Prisma relation mode?

Comment (andrewi-wd) [reply prisma/schema.prisma:5]: Added

**Review by kibertoad — APPROVED** [review]

**Pull request merged**